openlibrary/templates/languages/index.html

```diff
@@ -1,6 +1,6 @@
 $def with (languages)
```

Comment:
Mostly unrelated small fix here to make the title in the tab display the right thing.

```suggestion
$def with (languages)

$var title: _("Languages")
```